assertk-jvm/src/test/kotlin/test/assertk/assertions/AnyTest.kt

```diff
@@ -2,162 +2,167 @@ package test.assertk.assertions
 
 import assertk.assert
 import assertk.assertions.*
-import org.junit.jupiter.api.Nested
-import org.junit.jupiter.api.Test
+import org.junit.Test
+import test.assertk.assertions.AnyTest.nullableSubject
+import test.assertk.assertions.AnyTest.p
+import test.assertk.assertions.AnyTest.subject
+import test.assertk.assertions.AnyTest.unequal
 import kotlin.test.assertEquals
 import kotlin.test.assertFails
 
-class AnyTest {
-    private val p = AnyTest::class.java.`package`.name
+object AnyTest {
+    val p: String = AnyTest::class.java.`package`.name
+    val subject = BasicObject("test")
 
-    @Nested inner class `basic object` {
-        private val subject = BasicObject("test")
+    val nullableSubject: BasicObject? = BasicObject("test")
+    val unequal = BasicObject("not test")
+}
 
-        @Nested inner class `props` {
+class `AnyTest_basic_object` {
 
-            @Test fun `extracts jClass()`() {
-                assertEquals(BasicObject::class.java, assert(subject as TestObject).jClass().actual)
-            }
+    @Test
+    fun `props_extracts_jClass__`() {
```

Comment:
Extra underscores, find/replace issue?

Reply:
Kinda yes. The underscores replaced the special characters like `(` and `)` which where used here. I agree that it better to avoid trailing underscores but I wanted to discuss it with you :)
I'll change it.